### PR TITLE
feat(jest): 100% coverage for components/mixins/Touch

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖
self-explanatory

**Which issue(s) this PR fixes** 🔨

AP-544

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/mixins/Touch/index.test.ts [hunk 1]
```diff
@@ -68,14 +143,14 @@ describe('index.Touch.methods.sidebarSwipeHandler', () => {
...
-describe('index.Touch.created', () => {
+describe.skip('index.Touch.created', () => {
   test('0', () => {
     const result: any = index.Touch.created()
     expect(result).toMatchSnapshot()
   })
 })
 
-describe('index.Touch.methods.sidebarSwipeHandler', () => {
+describe.skip('index.Touch.methods.sidebarSwipeHandler', () => {
```
Comment (stavares843) [inline components/mixins/Touch/index.test.ts:153]: can we remove the tests if they are skipped?

File: components/mixins/Touch/index.test.ts [hunk 2]
```diff
@@ -7,7 +7,82 @@ describe('index.Touch.created', () => {
...
+        },
+      },
+      $device: {
+        isMobile: true,
+      },
+      sidebar: false,
+    }
+    const givenFunction: any = index.Touch.methods.sidebarSwipeHandler(local)
+    givenFunction('right')
+    // expect(local.sidebar).toBeTruthy()
```
Comment (stavares843) [inline components/mixins/Touch/index.test.ts:81]: commented out code 🔨

**Comment by stavares843** [comment]
minor comment

File: components/mixins/Touch/index.test.ts (hunk 2 above)
Comment (drepram) [reply components/mixins/Touch/index.test.ts:81]: The comment was actually intentional, to give a more informed decision on why I didn't do that (followed by the long comment below that line). Should I remove it?

**Review by phillsatellite — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**